docs/source/tutorials/getting-started/transfer.rst

```diff
@@ -324,8 +324,8 @@ implements the
 :ref:`Disclosure <type-daml-finance-interface-util-disclosure-disclosure-97052>`
 interface. This interface exposes the ``AddObservers`` and ``RemoveObservers`` choices, which can be
 used to disclose / undisclose Bob's account contract to Alice. In order to exercise these choices,
-you can use the :ref:`Account.exerciseInterfaceByKey <function-daml-finance-interface-account-account-exerciseinterfacebykey-13671>`
-utility function.
+you can use the :ref:`Account.exerciseInterfaceByKey
```

Comment:
does that render correctly with the linebreak in between?

Reply:
yes, double checked again